package.json

```diff
@@ -60,13 +60,13 @@
     "pelias-labels": "1.6.0",
     "pelias-logger": "0.2.0",
     "pelias-mock-logger": "^1.0.1",
-    "pelias-model": "4.6.0",
-    "pelias-query": "8.15.0",
+    "pelias-model": "4.7.1",
+    "pelias-query": "8.16.0",
     "pelias-text-analyzer": "1.8.0",
     "predicates": "^1.0.1",
     "retry": "^0.10.1",
     "request": "^2.79.0",
-    "stats-lite": "^2.0.4",
+    "stats-lite": "2.0.4",
```

Comment:
Maybe that was the only one with a carat that *also* had a newer version available?

As a matter of not getting blamed for problems, I think Greenkeeper never rewrites a version to include a carat since a lot of package maintainers like to play fast and loose with semver.